circuits/passportVerification/passportVerificationBuilder.circom

```diff
@@ -146,9 +140,15 @@ template PassportVerificationBuilder(
         passportVerificationFlow[i].encapsulatedContentHash <== encapsulatedContentHash[FLOW_MATRIX[i][4]-1];     //ec block num (-1 the same)
         passportVerificationFlow[i].signedAttributes        <== signedAttributes;
 
-        log("Flow result:", passportVerificationFlow[i].flowResult);
-
+        // log("Flow result:", passportVerificationFlow[i].flowResult);
+        if (i == 0) {
+            flowResultArray[i] <== passportVerificationFlow[i].flowResult;
+        } else {
+            flowResultArray[i] <== passportVerificationFlow[i].flowResult + flowResultArray[i-1];
+        }
     }
+
+    assert(flowResultArray[FLOW_MATRIX_HEIGHT-1] >= 1);
```

Comment:
Security problem. Use IsGreaterThan(). Assert does not add constraints

Reply:
done